### DISCUSSION TITLE
Scoring dataset runs, e.g. precision, recall, f-value

### DISCUSSION THREAD

**Discussion by yusuke-intern** [langfuse/langfuse#2511] [Ideas]
### Describe the feature or potential improvement

In LangSmith, for example, there is a feature to get precision, recall, f-value in experiment feature on a dataset.
https://docs.smith.langchain.com/how_to_guides/evaluation/evaluate_existing_experiment
Can we have similar feature in Langfuse?

**Comment by marcklingen** [comment]
thanks for sharing, evaluations on the dataset run level in langfuse would make a lot of sense. Creating an issue for this right now

**Reply by marcklingen** [reply]
https://github.com/langfuse/langfuse/issues/2512
